docs/docs/resources/editor_integration/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "yaml.schemas": {
+        "https://raw.githubusercontent.com/bakdata/kpops/main/schema_config.json": "config.yaml",
+        "https://raw.githubusercontent.com/bakdata/kpops/main/schema_pipeline.json": "pipeline.yaml"
```

Comment:
I tested by just changing the uploaded schema and tbh I didn't come to a conclusive result. I would assume that they would have thought of that considering that they allow using a URL like that.

As for the release pinning I couldn't find anything that would work. I can have the script generate a link to the latest release schema, but then it's up to the user to update it on their side.

We could have a whole new branch that is the same as the latest release, or we could make the CI upload the schemas to a schema store like [JSON Schema Store](https://www.schemastore.org/json/) when a release is published

Reply:
how about publishing the schemas to GitHub pages same as our docs (as we discussed on call)?

Reply:
Sounds great, but I suspect it is out of the scope of this issue, wdyt?

Reply:
implementing the CI part can be done in a followup. 
For now you could just add the two schema files to our docs manually so that they are published on gh-pages.
The urls should be something like https://bakdata.github.io/kpops/latest/schemas/pipeline.json and https://bakdata.github.io/kpops/latest/schemas/config.json
This way it will consistent and we don't have to change the URLs anymore